### PR TITLE
Initial Word2Vec backend class

### LINKED ISSUE #11: Testing
Tests can be run with `make test`

Schemas: #6 
Model: #6 
Endpoints: #10

### PR DESCRIPTION
**Pull request by christianjans**
Get the functions of the 4 functional requirements working with a test model.

I will provide the model files through our Discord. Just place them in a directory called `trained_models/test/` in the `word2med-backend/` directory, and then run `python3 word2med.py` and it should run the four functions on some test calls (defined in `word2med.py`).

TODO:
- [x] Ensure that the data structures returned by the methods can be put into the backend schema

When this is merged, I believe it should close:
- #1 
- #2 
- #3 
- #7

### CODE REVIEW

File: schemas.py
Comment (christianjans) [inline schemas.py:1]: I have the response schemas also return the input (i.e. the data in the query schemas). I wasn't sure if this is what we want or need. If not, let me know and I can remove it.

Comment (jboileau99) [reply schemas.py:1]: @christianjans I think this is good - contributes to the responses being as useable as possible and they don't have many fields in the first place so one extra field is fine

File: word2med.py [hunk 1]
```diff
@@ -1,12 +1,122 @@
...
+from typing import List, Tuple
+
+import numpy as np
+from sklearn.manifold import TSNE
+
+from models.word2vec import MalamudWord2Vec
+
 
 class Word2Med:
+    """The Word2Med model from a saved Word2Vec Gensim model"""
```
Comment (jboileau99) [inline word2med.py]:
```suggestion
    """The Word2Med model from a saved Word2Vec Gensim model
    
    Attributes:
        model_path: A string containing a path to the trained model
    """
```

Comment (jboileau99) [reply word2med.py]: following https://google.github.io/styleguide/pyguide.html#384-classes

**Review by jboileau99 — CHANGES_REQUESTED** [review]
Looks great! Good to merge whenever

**Comment by jboileau99** [comment]
@christianjans Just two small things - I added an env variable to specify the model path then passed that to the model constructor in the flask app, and ran `make docs` to update the api documentation.

**Pull request merged**